subprojects/gradle-guides-plugin/src/main/java/org/gradle/docs/guides/internal/GuidesDocumentationPlugin.java

```diff
@@ -184,7 +193,8 @@ private TaskProvider<? extends Task> renderGuidesDocumentation(TaskContainer tas
 
             // TODO: This is specific to guides
             attributes.put("imagesdir", "images");
-            attributes.put("stylesheet", null);
+// FIXME: null value for attributes seems to be broken right now.
+//            attributes.put("stylesheet", null);
```

Comment:
Does this affect the generated files at all?

Reply:
Not sure. Maybe it uses a default stylesheet if this is not set to `null`? Where could I see the generated files before and after?